Mathlib/MeasureTheory/Measure/OpenPos.lean

```diff
@@ -251,3 +253,35 @@ theorem measure_closedBall_pos (x : X) {r : ℝ≥0∞} (hr : r ≠ 0) : 0 < μ
 #align emetric.measure_closed_ball_pos EMetric.measure_closedBall_pos
 
 end EMetric
+
+section MeasureZero
+/-! ## Meagre sets and measure zero
+In general, neither of meagre and measure zero implies the other.
+- for all $α ∈ (0,1)$, there is a generalised Cantor set $C ⊆ [0,1]$ of measure `α`.
+Cantor sets are nowhere dense (hence meagre). Taking a countable union of fat Cantor sets whose
+measure approaches 1 even yields a meagre set of measure 1.
```

Comment:
The complement of the set of Liouville numbers is another counterexample, isn't it?